ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-presubmits.yaml

```diff
@@ -16,6 +16,285 @@ presubmits:
     - gh-pages
     - release-1.20
     trigger: (?m)^/test( | .* )kata-containers,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build04
+    context: ci/prow/ci-critest
```

Comment:
I still think this hsould be `critest-el8`, but am willing to let that be a follow up if needed